CoreVisualSchema/src/au/gov/asd/tac/constellation/graph/schema/visual/compatibility/VisualSchemaV6UpdateProvider.java

```diff
@@ -28,8 +28,8 @@
 /**
  *
  * This update ensures that the node_labels_top and node_labels bottom graph
- * attributes appropriately change name to node_top_labels_colours and
- * node_bottom_labels_colours
+ * attributes appropriately change name to node_top_labels_colors and
+ * node_bottom_labels_colors
```

Comment:
in this case you'll actually want to retain "colours" as that is what it was set to